### ISSUE TITLE
Github Actions: logs visualization

### ISSUE THREAD

**Issue by Nek** [spotify/backstage#1858]
The Github Actions plugin should be able to show logs for a specific workflow run.

**Comment by NetPenguins** [comment]
Interested in helping on this one. Are there any specifics you had in mind? I see there being a call to the REST api to get the workflows and then assign them to a list of sorts ..... then when selected make a call with the associated ID to obtain the log for that run. Any concerns with that approach?

**Comment by stefanalund** [comment]
Thanks @NetPenguins -- I've assigned you. @Nek any further clarifications to provide?

**Comment by Nek** [comment]
@NetPenguins take a look at https://github.com/spotify/backstage/tree/master/plugins/circleci. There is a component for logs visualisation that can be a good fit for these logs too.

**Comment by Nek** [comment]
@NetPenguins I think starting simple is the best way to go.

**Comment by NetPenguins** [comment]
@Nek Do you have a repo that has multiple jobs per run? It would also be helpful to have a job/run that has had a failure in it to test the styling around failures.

**Comment by NetPenguins** [comment]
@Nek So I realized when I dug in that I had an very outdated version. So after upgrading I also noticed that there is now a Jobs element inside the WorkFlowRunDetails page. The only thing I can think to add that would create a complete log would be the steps per job. Is this what you were after in the initial request? If so I will add the nested steps under their respective jobs using the same formatted tree that displays the jobs.
